src/routes.js

```diff
@@ -21,6 +22,7 @@ const routes = (props) => {
           <Switch>
             {authStatus ? <Route path="/articles/:slug" exact component={Article} /> : null}
             {authStatus ? <Route path="/createArticle" exact component={CreateArticle} /> : null}
+            <Route path="/profile" exact component={Profile} />
```

Comment:
@Seryazi-Andela, I think the profile editing route should be protected and only viewable by authorized users.